src_c/doc/font_doc.h

```diff
@@ -10,6 +10,7 @@
 #define DOC_PYGAMEFONTSYSFONT "SysFont(name, size, bold=False, italic=False) -> Font\ncreate a Font object from the system fonts"
 #define DOC_PYGAMEFONTFONT "Font(filename, size) -> Font\nFont(pathlib.Path, size) -> Font\nFont(object, size) -> Font\ncreate a new Font object from a file"
 #define DOC_FONTBOLD "bold -> bool\nGets or sets whether the font should be rendered in (faked) bold."
+#define DOC_FONTNAME "name -> str\nGets the font's name."
```

Comment:
```suggestion
#define DOC_PYGAMEFONTNAME "name -> str\nGets the font's name."
```
Annoyingly the freetype module already stole the `DOC_FONTNAME` macro name

Reply:
This file is autogenerated, so I'm not sure how we'd get around that.

Reply:
Figured it out: you need to add it to the `#undef` lines in static.c